backend/backend/urls.py

```diff
@@ -63,10 +63,26 @@
 ]
 
 try:
-    import pluggable_apps.subscription.urls  # noqa: F401
 
     urlpatterns += [
         path("", include("pluggable_apps.subscription.urls")),
     ]
 except ImportError:
     pass
+
+try:
+
+    urlpatterns += [
+        path("", include("pluggable_apps.public_shares.share_manager.urls")),
+    ]
+except ImportError:
+    pass
+
+try:
+    import pluggable_apps.clone.urls  # noqa: F401
```

Comment:
Do we need this? Because I noticed something similar is done for subscription app and it is removed in line 66.

Reply:
This will be needed since we need to add the `clone` app to the list of Urls.